config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-main.yaml

```
@@ -193,8 +196,8 @@ presubmits:
             value: "3.4.13-0"
           - name: COREDNS_VERSION_UPGRADE_TO
             value: "1.7.0"
-          - name: GINKGO_FOCUS
-            value: "\\[Periodic-K8SVersion\\]"
+          - name: GINKGO_SKIP
+            value: "\\[PR-Blocking\\] \\[K8s-Upgrade\\]"
```

```suggestion
        - name: GINKGO_FOCUS
           value: "\\[K8s-Upgrade\\]"
```
The intent of this job is to give the developers the possibility to test a full upgrade + conformance with the current PR (without waiting for merge and the periodic job to run)